test/unit/range/container/small_string_test.cpp

```diff
@@ -111,6 +111,21 @@ TEST(small_string, implicit_conversion)
     EXPECT_EQ(str, "hello"s);  // explicit
 }
 
+TEST(small_string, erase)
+{
+    small_string em{"hello"};
+    em.erase();
+    EXPECT_EQ(em.str(), ""s);
```

Comment:
Can you please add a constexpr version of the test?

Reply:
sure